src/analysis/normed_space/add_torsor.lean

```diff
@@ -301,6 +308,30 @@ end
 
 end normed_space
 
+namespace affine_map
+
+variables {𝕜 V₂ P₂ : Type*} [normed_field 𝕜] [semi_normed_space 𝕜 V] [semi_normed_group V₂]
+variables [semi_normed_space 𝕜 V₂] [pseudo_metric_space P₂] [semi_normed_add_torsor V₂ P₂]
+include V₂
+
+lemma continuous_of_normed_iff (f : P →ᵃ[𝕜] P₂) : continuous f ↔ continuous f.linear :=
```

Comment:
I would expect a name of the form `continuous_iff_continuous_linear`.

Reply:
I have just discovered that this lemma already exists under the name `continuous_linear_iff` (in the same `affine_map` namespace) so I have dropped mine https://github.com/leanprover-community/mathlib/blob/fefd8a38be7811574cd2ec2f77d3a393a407f112/src/analysis/normed_space/affine_isometry.lean#L543